.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for a new icepyx feature
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
```

Comment:
Should we make the input format consistent (i.e. make these input boxes as well, where presumably the instructions are temporary text or separate from the response without the user manually deleting)?